composer.json

```diff
@@ -23,38 +23,41 @@
     "require": {
         "php": "^7.2",
         "doctrine/doctrine-bundle": "^1.12 || ^2.0",
-        "doctrine/persistence": "^1.3",
+        "doctrine/persistence": "^1.3.4",
         "sonata-project/datagrid-bundle": "^2.5",
         "sonata-project/doctrine-extensions": "^1.5",
         "sonata-project/form-extensions": "^0.1 || ^1.4",
         "symfony/config": "^4.4",
         "symfony/console": "^4.4",
-        "symfony/dependency-injection": "^4.4",
+        "symfony/dependency-injection": "^4.4.3",
```

Comment:
I would avoid this change if not completely necessary, to make consistent will all bundles requiring just ^4.4

Reply:
> I bump some package to version from `composer update --prefer-lowest`, result:
> 
> * [before: 10 min 16 sec](https://travis-ci.org/github/sonata-project/SonataNotificationBundle/builds/711389975)
> * [after:  4 min 6 sec](https://travis-ci.org/github/sonata-project/SonataNotificationBundle/builds/711394598)

I want avoid timeout for ``--prerfer-lowest` tests.  Lets check it for composer without patch version: https://travis-ci.org/github/sonata-project/SonataNotificationBundle/builds/711399641

Reply:
4 min 17 sec :smiley:

Reply:
this concrete changes makes all the differente? 🤔